app/routes/users/components/StudentConfirmation.js

```diff
@@ -119,7 +119,7 @@ const StudentConfirmation = ({
 };
 
 const validate = createValidator({
-  studentUsername: [required()],
+  studentUsername: [isNotEmail('Kun NTNU-brukernavn')],
```

Comment:
couldn't we just do `input.replace('@stud.ntnu.no', '') or something before passing it to the API instead?